index.html

```diff
@@ -28,9 +28,48 @@
             </ul>
         </nav>
         <main class="main">
+            <!-- --------avatar-component-------------- -->
             <section id="avatar" class="section">
                 <header class="section-header">Avatar</header>
-                <article class="container-info"></article>
+                <article class="container-info">
+                    <div class="info">
+                        <h4> Avatar is used to represent a user or Profile picture</h4></div>
+                    <div class="avatar-container">
+                        <div class="text-badge">Default Avatar</div>
+                        <div class="avatar">
+                            <div class="image">
+                                <img src="image.jpg" alt="sunset" class="avatar-image">
+                            </div>
+                        </div>
+                        <div class="code">
+                            <iframe src="https://www.thiscodeworks.com/embed/61f8e6601e1df40015379388" style="width: 100%; height: 151px;" frameborder="0"></iframe>
+                        </div>
+                    </div>
+                    <div class="avatar-container">
+                        <div class="text-badge">Circular Avatar</div>
+                        <div class="avatar">
+                            <div class="image">
+                                <img src="image.jpg" alt="sunset" class="avatar-image img-round">
```

Comment:
at one place you are giving the class name as avatar-image and in another one giving img-round. It should be consistent, better to have both as img

Reply:
I have changes the class name to round-image and also similar changes made